### DISCUSSION TITLE
failed to start / install servers

### DISCUSSION THREAD

**Discussion by 1Euro7Cent** [pterodactyl/panel#3062] [Support]
I was assignt to the discussion thingy instead of the issue tab

I know. Go to discord. But in discord I do not get any response.

I cannot install / start a server. I can install the server without the egg thingy when I load a server in there the server doesn't start
when I try to start a server
log: https://ptero.co/arepeyifuv
console from a not starting server: (the server worked on my local machine and on the server before pterodactyl) https://cdn.discordapp.com/attachments/764995848101691402/805534384190259301/unknown.png

**Comment by parkervcp** [comment]
I would ask that you get a `docker info` and `uname -a`
